source/common/http/http1/codec_impl.cc

```diff
@@ -507,6 +515,9 @@ Status ConnectionImpl::completeLastHeader() {
   ASSERT(dispatching_);
```

Comment:
optional as it's not your code, but consider renaming completeLastHeader as I read it as completing _the_ last header, not completing the most recent header to be processed

Reply:
Will do in a separate PR

Reply:
Done in PR #22408